jscomp/bsb/bsb_ninja_file_groups.ml

```diff
@@ -237,13 +237,18 @@ let handle_files_per_dir
 
   if not (Bsb_file_groups.is_empty group) then begin
     Buffer.add_string buf "(subdir ";
+    Buffer.add_char buf '"';
     Buffer.add_string buf rel_group_dir;
+    Buffer.add_char buf '"';
     Buffer.add_char buf '\n';
     if group.subdirs <> [] then begin
       Buffer.add_string buf "(dirs :standard";
       Ext_list.iter group.subdirs (fun subdir ->
         Buffer.add_char buf ' ';
-        Buffer.add_string buf subdir);
+        Buffer.add_char buf '"';
+        Buffer.add_string buf subdir;
+        Buffer.add_char buf '"';
```

Comment:
I just pushed a commit to this branch exposing this function. Would be cool to use it instead.

```suggestion
        Buffer.add_string buf (Ext_filename.maybe_quote_for_dune subdir);
```